.github/workflows/build_and_test.yml

```diff
@@ -22,9 +22,13 @@ jobs:
           toolchain: nightly
       - uses: Swatinem/rust-cache@v1
       - name: install mdbook
-        run: cargo install mdbook
+        uses: baptiste0928/cargo-install@v1.3.0
```

Comment:
Why depend on a random person's workflow for a single line?

Reply:
the workflow contains cache for it, otherwise `cargo install cargo-make` will take more than 10+ mins